pkg/kubectl/cmd/version.go

```diff
@@ -17,17 +17,26 @@ limitations under the License.
 package cmd
 
 import (
+	"errors"
 	"fmt"
 	"io"
 
+	"encoding/json"
```

Comment:
ha, i totally missed that during moving things around. thanks for the catch. Could you give this pr and lgtm? i got this dropped after an automatic retest.